package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "oag-template",
+	"version": "1.0.0",
+	"private": true,
+	"description": "OAG template using Jekyll website and GCWeb theme",
+	"main": "index.html",
+	"scripts": {
+		"lint:css": "stylelint '**/*.css'",
+		"format:js": "prettier --write '**/*.js'",
+		"postbuild": "prettier --write _site/*.html",
+		"build-css": "postcss src/assets/css/oag.css --use postcss-nested -o dist/assets/css/oag.min.css"
+	},
+	"devDependencies": {
+		"autoprefixer": "^10.4.20",
+		"bower": "^1.8.14",
```

Comment:
Do you need it?

```suggestion
```

Regarde le package.json dans wet-boew. Je pense qu'on l'a enlever récemment.

Reply:
removed